corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -364,10 +364,9 @@ def update_log(action, doc_ids):
             couch_class.wrap(doc)._migration_sync_to_sql(obj, save=False)
         if creates or updates:
             with transaction.atomic(), disable_sync_to_couch(sql_class):
-                sql_class.objects.bulk_create(creates)
-                if updates:
-                    for obj in updates:
-                        obj.save()
+                sql_class.objects.bulk_create(creates, ignore_conflicts=True)
```

Comment:
Reading the docs (https://docs.djangoproject.com/en/3.2/ref/models/querysets/#bulk-create) my interpretation is that this flag will keep what's already in the db and discard the insert for any duplicate row. Sounds like what you want, just spelling it out for my understanding. I'm not sure I understand the race condition at a detailed enough level, but it sounds like the logic is that if the row is already there, then the mechanism keeping it up to date is working, so you'd never want to replace it with the "new" version in the `bulk_create`.

Reply:
I am sure this should not be an issue but flagging that `ignore_conflicts` with bulk create is only available after Postgres 9.5.

Reply:
That should be fine. We announced upgrading to 9.6 and dropped support for 9.5 years ago, so this shouldn't be an issue.

Reply:
> this flag will keep what's already in the db and discard the insert for any duplicate row. Sounds like what you want
> ...
> if the row is already there, then the mechanism keeping it up to date is working, so you'd never want to replace it with the "new" version in the `bulk_create`.

Correct. Although `bulk_create` will never replace/overwrite, it raises an error which previously caused the migration to crash.